terminus/tests/road_test.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from geometry.point import Point
+from geometry.latlon import LatLon
+from models.road import Road
+from models.street import Street
+import math
+
+
+class RoadTest(unittest.TestCase):
+
+    def test_get_waypoint_positions(self):
+        points = [Point(0.0, 1.0), Point(1.0, 1.0), Point(2.0, 1.0)]
```

Comment:
When working with collections it is a good practice to test also the empty case.